common/djangoapps/edxmako/shortcuts.py

```diff
@@ -55,7 +55,7 @@ def marketing_link(name):
             return reverse(link_map[name])
     else:
         log.debug("Cannot find corresponding link for name: %s", name)
-        return '#'
+        return '/'
```

Comment:
Using `a href="#"` is invalid so in instances where the link isn't meant to do anything, we're sending users to the root/homepage.